doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
 +
 image::cases/case-management-invalid-spec.png[]
 +
-Click *Complete*. 
+Click *Complete*.
 +
-Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`. 
-. In Showcase, click *Refresh* in the upper-right corner. Notice that a `Prepare hardware task` appears in the *Completed* column and another appears in the *In Progress* column. 
+Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`.
```

Comment:
Consider something like:
The value in the  *Task Inbox* for the `Prepare hardware spec` is 'Ready' instead of 'Complete'.